packages/commonwealth/scripts/load-db.sh

```diff
@@ -18,12 +18,12 @@ if [ "$1" ]; then
   DUMP_NAME=$1
 fi
 
-psql -d commonwealth -U commonwealth -f "$DUMP_NAME";
+psql -h localhost -d commonwealth -U commonwealth -f "$DUMP_NAME";
 
 if [ "$ETH_ALCHEMY_API_KEY" ]
 then
   ETH_ALCHEMY_URL="eth-mainnet.g.alchemy.com/v2"
-  psql -d commonwealth -U commonwealth -c "UPDATE \"ChainNodes\" SET url = 'wss://$ETH_ALCHEMY_URL/$ETH_ALCHEMY_API_KEY', alt_wallet_url = 'https://$ETH_ALCHEMY_URL/$ETH_ALCHEMY_API_KEY' WHERE eth_chain_id = 1;"
+  psql -h localhost -d commonwealth -U commonwealth -c "UPDATE \"ChainNodes\" SET url = 'wss://$ETH_ALCHEMY_URL/$ETH_ALCHEMY_API_KEY', alt_wallet_url = 'https://$ETH_ALCHEMY_URL/$ETH_ALCHEMY_API_KEY' WHERE eth_chain_id = 1;"
```

Comment:
@jnaviask is it ok to add `-h localhost` to these scripts?
does it break anything?

Reply:
Will be fine as this script is only run locally / against local postgres databases.

Reply:
great, lets keep it then